src/rules/typescript.ts

```diff
@@ -2,13 +2,15 @@
 // These rules find errors related to TypeScript features
 // Reference https://palantir.github.io/tslint/rules/
 
-module.exports = {
+import {typedefWhitespaceLeft, typedefWhitespaceRight} from './rule-helpers/typedef-whitespace';
+
+export const typescriptRules = {
   // Enforces function overloads to be consecutive.
   'adjacent-overload-signatures': true,
   // Requires explicit visibility declarations for class members.
   'member-access': false,
   // Enforces member ordering.
-  'member-ordering': [true, {'order': require('./rule-helpers/member-ordering-list')}],
+  // 'member-ordering': [true, {'order': memberOrderingRule}],
```

Comment:
Thanks for this catch as well. 
When I was doing migrations from `require()` to `import`/`export`, I forgot to uncomment this back in.